Source/JavaScriptCore/fuzzilli/Fuzzilli.h

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright (C) 2023 Apple Inc. All rights reserved.
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Library General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Library General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Library General Public License
+ *  along with this library; see the file COPYING.LIB.  If not, write to
+ *  the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ *  Boston, MA 02110-1301, USA.
+ *
+ */
+
+#pragma once
+
+#include <fcntl.h>
+#include <wtf/Vector.h>
+
+#if ENABLE(FUZZILLI)
+
+#if ENABLE(LIBFUZZER)
+#error "Fuzzilli requires libFuzzer to be disabled!"
+#endif
+
+namespace Fuzzilli {
+
+struct SharedData {
+    uint32_t numEdges;
+    uint8_t edges[];
+};
+
+extern struct SharedData* sharedData;
+
+extern uint32_t* edgesStart;
+extern uint32_t* edgesStop;
+
+extern char* reprlInputData;
+extern size_t numPendingRejectedPromises;
+
+void resetCoverageEdges();
+
+FILE* logFile();
```

Comment:
Can we use FilePrintStream instead? This can make WebKit classes used without various hacks.